platform/platform-app/src/main/resources/features.xml

```diff
@@ -902,6 +902,28 @@
             mvn:ddf.platform.solr/platform-solr-server-standalone/${project.version}/txt/synonyms
         </configfile>
 
+        <configfile finalname="/data/solr/attributes/conf/solr.xml">
```

Comment:
The persistence store is in platform which is why I thought the solr configurations were specified here.  If these should go in catalog, then should the persistence store move to catalog?